src/components/Addbook.js

```diff
@@ -8,36 +8,33 @@ import { addBook } from '../redux/books/books';
 const AddBook = (props) => {
   const { categories } = props;
   const dispatch = useDispatch();
-  const booksLength = useSelector((state) => state.books.length);
+  const booksKeys = useSelector((state) => Math.max(...state.books.map((book) => book.item_id)));
 
   const submitBookToStore = () => {
     const bookTitle = document.querySelector('#bookTitle');
-    const bookAuthor = document.querySelector('#bookAuthor');
 
```

Comment:
[ OPTIONAL ] The book item should have a title and author. Users can only add title when creating a new book for now.